gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopOutputStreamIntegrationTest.java

```diff
@@ -130,7 +130,7 @@ public void time_statistics() throws Exception {
     fout.close();
 
     TestUtils.verifyDurationMetric(stats, GhfsStatistic.STREAM_WRITE_CLOSE_OPERATIONS, 1);
-    TestUtils.verifyDurationMetric(stats, GhfsStatistic.STREAM_WRITE_OPERATIONS, 4);
+    TestUtils.verifyDurationMetric(stats, GhfsStatistic.STREAM_WRITE_OPERATIONS, 2);
```

Comment:
why it reduced?

Reply:
there was a bug in the verification code. (Plz refer TestUtils.java). I fixed that. The correct value should have been 2. One for each write() in the test.